control-plane/mbus-api/Cargo.toml

```diff
@@ -7,9 +7,7 @@ edition = "2018"
 [dependencies]
```

Comment:
We can probably move this `mbus-api` crate to top level? It seems to be the only child of `control-plane/`.

Reply:
Yeah good point, there's no more control-plane.

Reply:
Done.